lib/components/settings/settings.dart

```diff
@@ -274,93 +281,93 @@ class _SettingsTileData {
   });
 }
 
-List<_SettingsTileData> _settingsTiles = [
-  //Search HEADER
-  const _SettingsTileData(
-    title: "Search",
-  ),
-  //Connectivity HEADER
-  _SettingsTileData(
-    title: LSX.settings.headersConnectivity,
-  ),
-  //Connectivity TILES
-  _SettingsTileData(
-    title: LSX.settings.pagesNetworkTitle,
-    subtitle: LSX.settings.pagesNetworkSubtitle,
-    icon: Icons.wifi,
-    page: const SettingsPageNetwork(),
-  ),
-  _SettingsTileData(
-    title: LSX.settings.pagesConnectionsTitle,
-    subtitle: LSX.settings.pagesConnectionsSubtitle,
-    icon: Icons.devices_rounded,
-    page: const SettingsPageConnectedDevices(),
-  ),
-  //Personalize HEADER
-  _SettingsTileData(
-    title: LSX.settings.headersPersonalize,
-  ),
-  //Personalize TILES
-  _SettingsTileData(
-    title: LSX.settings.pagesCustomizationTitle,
-    subtitle: LSX.settings.pagesCustomizationSubtitle,
-    icon: Icons.color_lens_outlined,
-    page: const SettingsPageCustomization(),
-  ),
+List<_SettingsTileData> get _settingsTiles => <_SettingsTileData>[
```

Comment:
We'll just have to have this for now but in future we should consider another approach possibly